frontend/app/src/i18n/en.json

```diff
@@ -687,7 +687,30 @@
         "storageIndexAction": "Storage Index Actions",
         "cyclesDispenserAction": "Cycles Dispenser Actions",
         "registryAction": "Registry Actions",
-        "unknownActionCategory": "Other Actions"
+        "unknownActionCategory": "Other Actions",
+        "makeProposal": "Make proposal",
+        "maker": {
+            "header": "Make a proposal",
+            "title": "Title",
+            "summary": "Summary",
+            "url": "Url",
+            "enterTitle": "Enter a title",
+            "enterSummary": "Enter a summary",
+            "enterUrl": "Enter an optional url",
+            "summaryRules": "supports markdown",
+            "urlRules": "link to supplemental details",
+            "unexpectedError": "Unexpected error submitting proposal",
+            "toPrincipal": "Recipient account principal",
+            "enterToPrincipal": "Enter a recipient account principal",
+            "amount": "Amount",
+            "amountRules": "at least 1 {token} token",
+            "enterAmount": "Enter a quantity of {token} tokens",
+            "type": "Proposal type",
+            "treasury": "Token to transfer from treasury",
+            "insufficientFunds": "You need at least {min_balance} {token} tokens in your wallet to submit a proposal",
+            "info": "It costs 4 {token} tokens to submit a proposal. This is to prevent spam. Only if your proposal is adopted will this be refunded. It can take up to an hour after your proposal is adopted before you receive the refund.",
```

Comment:
It is 4 CHAT but it won't be 4 of other tokens. Also, it won't take up to an hour, it'll be ~1 minute since we already poll every governance canister every minute.

Reply:
Well it is quite CHAT specific anyway at the moment - I was planning on generalising later